sdk/python/tests/test_materialize_from_bigquery_to_datastore.py

```diff
@@ -50,10 +52,18 @@ def test_bigquery_table_to_datastore_correctness(self):
         job = self.client.load_table_from_dataframe(df, table_id, job_config=job_config)
         job.result()
 
+        # create Entity
+        e = Entity(
+            name="Driver ID",
+            description="id for driver",
+            join_key="driver_id",
+            value_type=ValueType.INT32,
+        )
+
         # create FeatureView
         fv = FeatureView(
             name="test_bq_table_correctness",
-            entities=["driver_id"],
+            entities=["Driver ID"],
```

Comment:
Wait, we support arbitrary text for the name? I don't think thats a good idea. We should probably just be supporting lowercase alphanumeric with underscore.